.github/workflows/check.yml

```diff
@@ -206,6 +206,8 @@ jobs:
                   args: evcxr_repl
 
             - name: Install cargo-generate
+              # BACKCOMPAT: zeroize dependency requires Rust 51.0 or newer
+              if: matrix.os != 'windows-latest' && matrix.rust-version >= '1.51.0'
```

Comment:
Why `windows` builds are ignored?

Reply:
Well, I didn't quite understand why they are ignored for Evcxr, and just copied to avoid any possible trouble

Reply:
Tests for `evcxr` are ignored because at the beginning our repl integration didn't work on Windows.
So it's not related to your case

Reply:
BTW @dima74 I suppose we can enable repl tests for windows now